ores/scorer_models/rev_id_scorer.py

```diff
@@ -0,0 +1,51 @@
+from revscoring import ScorerModel
+from revscoring.datasources import revision
+from revscoring.features import Feature
+
+
+def process_last_two_in_rev_id(rev_id):
+    return int(str(rev_id)[-2:])
+
+last_two_in_rev_id = Feature(
+    "revision.last_two_in_rev_id",
+    process_last_two_in_rev_id,
+    returns=int,
+    depends_on=[revision.id]
+)
+
+
+class RevIdScorer(ScorerModel):
+    """
+    Implements a basic, testing scorer that predicts whether a revision ID's
+    last two digits are greater than 50.
+    """
+    def __init__(self, version=None):
+        super().__init__([last_two_in_rev_id], version=version)
+
+    def score(self, feature_values):
+        probability = feature_values[0] / 100
```

Comment:
one of them should be float otherwise it gives zero ( 1 / 10 = 0 but 1/10.0 = 0.1)